src/NServiceBus.Core.Tests/API/APIApprovals.cs

```diff
@@ -16,7 +16,7 @@ public class APIApprovals
         [MethodImpl(MethodImplOptions.NoInlining)]
         public void ApproveNServiceBus()
         {
-            var assemblyPath = Path.GetFullPath(typeof(IBusSessionFactory).Assembly.Location);
+            var assemblyPath = Path.GetFullPath(typeof(IBusSession).Assembly.Location);
```

Comment:
I don't now why exactly `IBusSessionFactory` ended up as the "NSB.Core marker interface" but one of the endpoint interfaces/classes would seem more appropriate to me, e.g.`IEndpointInstance` or `Endpoint`?